README.md

```diff
@@ -2,20 +2,11 @@
 
 [![Code Climate](https://codeclimate.com/github/Jellyvision/lateralus/badges/gpa.svg)](https://codeclimate.com/github/Jellyvision/lateralus)
 
-![Lateralus logo](img/logo.png)
-
 Lateralus is a single page web application framework.  It is designed around the idea that UIs should be divided into discreet, reusable, loosely-coupled components.  These components should communicate only indirectly and be easily swappable or removable.  Lateralus provides patterns and utilities to cleanly isolate individual components of your app.
 
-**[Link: A simple demo app built with Lateralus](https://github.com/Jellyvision/marker)**
-
-## Dependencies (Core framework)
+Lateralus is built upon Backbone, Underscore, jQuery, and Mustache.  These dependencies are baked into the standard build, so you do not need to provide them in your app.
```

Comment:
It just occurred to me that this is no longer true — I wound up having to externalize the dependencies to stay compatible with existing downstream projects.

I will update this tomorrow; but the rest of this PR can still be reviewed.